src/test/test_utils.hpp

```diff
@@ -298,6 +308,24 @@ class ConstructorEnabledModelManager : public ovms::ModelManager {
     }
 };
 
+class RequestMockModelInstance : public ovms::ModelInstance {
```

Comment:
Rename to MockedMetadataModelInstance? Or sth similar to know from name of the class what is mocked